src/vis/vis.js

```diff
@@ -416,6 +362,10 @@ var Vis = View.extend({
     // Lastly: reset the layer models on the map
     this.map.layers.reset(layerModels);
 
+    // Global variable for easier console debugging / testing
+    window.vis = this;
+    window.map = map;
```

Comment:
While I understand the intents of exposing this (like explained in the comment just above), setting these directly on window could possibly overwrite someone's vars (that's using cartodb.js for a standalone app), do we really want to expose them here?